src/insert.c

```diff
@@ -33,6 +33,8 @@
 #include "insert_chunk_state.h"
 #include "insert_statement_state.h"
 
+void add_number_tuples_processed(uint64);
```

Comment:
I prefer a .h file even if it is small. Not sure where we landed on that though.